packages/sheets-ui/src/views/permission/panel-detail/index.tsx

```diff
@@ -329,15 +329,16 @@ export const SheetPermissionPanelDetail = ({ fromSheetBar }: { fromSheetBar: boo
         sheetPermissionPanelModel.setRule(rule);
     };
 
-    const handlePanelClick = (e: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
+    const handlePanelClick = (e: MouseEvent) => {
         const handleOutClick = rangeSelectorActionsRef.current?.handleOutClick;
-        handleOutClick && handleOutClick(e, isFocusRangeSelectorSet);
+        handleOutClick && handleOutClick(e, () => isFocusRangeSelectorSet(false));
     };
 
+    useSidebarClick(handlePanelClick);
```

Comment:
This hook is suspicious. What about other focusable elements not in the panel, especially considering we are an SDK?

Reply:
This focused state is left entirely to the external business of the component (different cases have different defocusing moments)